src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -99,4 +121,15 @@ public static function checkForIllegalCsvPropertyKeys($propertyKey)
             throw new BadPropertyKeyException($propertyKey);
         }
     }
+
+    /**
+     * @param string $attributeName Attribute name to output in exception.
+     * @param string $attributeValue Attribute value to check if it exceeds character limit.
+     */
+    public static function checkValueNotExceedingCharacterLimit($attributeName, $attributeValue)
```

Comment:
Since this check is specific to attribute values, the method name can be more concise.

Reply:
Done :heavy_check_mark: